src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -37,19 +37,22 @@
  * @see EmailWrapper
  */
 public final class EmailGenerator {
+    // feedback action strings
```

Comment:
Is there any reason why you moved "feedback action strings" up? Try not to move things around so we can keep track of  changes. Perhaps move it back down if there isn't a good reason

Reply:
The reason why I had to shift it up was because it was referenced in status-related strings. There would be a `illegal forward reference` compile error if I used the original positions. Hope that clear things up!

Reply:
I overlooked this, thanks for letting me know :)